pkgs/top-level/all-packages.nix

```diff
@@ -15244,8 +15244,12 @@ in
 
   labelImg = callPackage ../applications/science/machine-learning/labelimg { };
 
+  mailman = python3.pkgs.callPackage ../servers/mail/mailman/wrapped.nix { };
```

Comment:
Use normal `callPackage`, and have `python3` as parameter instead of the individual Python packages.